### PR TITLE
clifm: 1.17 -> 1.18

### PR DESCRIPTION
**Pull request by nadir-ishiguro**
## Description of changes

Changelog: https://github.com/leo-arch/clifm/releases/tag/v1.18

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by AndersonTorres — CHANGES_REQUESTED** [review]
Please rewrite makeFlags as

```
makeFlags = [
  "PREFIX=${placeholder "out"}"
  "DATADIR=${placeholder "out"}/share"
];
```

since DESTDIR is not recommended for Nixpkgs.

**Comment by nadir-ishiguro** [comment]
Thank you, done. 

Unrelated question: Is it fine to reformat packages with the now official formatter in the course of updates like this? With a separate commit I mean.

**Comment by AndersonTorres** [comment]
> Unrelated question: Is it fine to reformat packages with the now official formatter in the course of updates like this? With a separate commit I mean.

100% fine.

**Review by drupol — CHANGES_REQUESTED** [review]
Please update the commit log message as such:

`clifm: format with nixfmt-rfc-style`

**Comment by nadir-ishiguro** [comment]
> Please update the commit log message as such:
> 
> `clifm: format with nixfmt-rfc-style`

Of course. Thank you for pointing that out.

File: pkgs/applications/file-managers/clifm/default.nix [hunk 1]
```diff
@@ -1,22 +1,36 @@
-{ stdenv, lib, fetchFromGitHub, libcap, acl, file, readline, python3 }:
+{
+  stdenv,
```
Comment (AndersonTorres) [inline pkgs/applications/file-managers/clifm/default.nix:2]: Is that atrocity the end result of RFC? Look how they massacred the set.
My apologies...

**Review by AndersonTorres — APPROVED** [review]

**Review by drupol — APPROVED** [review]

**Pull request merged**